segmentio/golines
# More code moving

- Fix links to test data directory in README

- Move generate out of top level

    Move this into a new directory, `scripts` to be a bit more conventional

- `pre-commit` use full linter for golangci-lint

    This runs the linter across all files which can be slower, but should be
    improved with usage of its cache. The benefit here is that all the
    linters (including ones that need to read all files) can be run each
    time, which is how it works in CI.

- Update make rules

    * Remove some unused ones
    * Update any that were impacted by recent changes

- Add contributing guide
